index.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset='utf-8'>
+    <meta http-equiv='X-UA-Compatible' content='IE=edge'>
+    <meta name='viewport' content='width=device-width, initial-scale=1'>
+
+    <title></title>
+    <link rel='stylesheet' type='text/css' media='screen' href='style.css'>
+    <!--script src='main.js'></script-->
+</head>
+<body>
+    <header class="toolbar">
+        <img id="img1" src="image_geometry_1.svg" alt="">
+        <div class="ic_icon">
+          <button id="btn"></button>
+          <button id="btn"></button>
+          <button id="btn"></button>
+        </div>
```

Comment:
- [ ] Nice work using `header` tag for the "Toobar" section :+1: However it would be nice to wrap it in a `nav` tag for semantics reason.